readthedocs/api/v3/permissions.py

```diff
@@ -17,7 +17,7 @@ def has_permission(self, request, view):
             return True
 
 
-class PublicDetailPrivateListing(IsAuthenticated):
+class PublicDetailPrivateListing(BasePermission):
```

Comment:
this class is overriding the has_permission method, and not calling super(), so it doesn't matter. We already have a check for auth users in another class

https://github.com/readthedocs/readthedocs.org/blob/4a19a9e668008fd65ee33fffe2471299ab1db492/readthedocs/api/v3/permissions.py#L76-L93

Reply:
Confirmed.